pandas/_libs/tslibs/offsets.pyx

```diff
@@ -3627,24 +3629,91 @@ cdef class CustomBusinessHour(BusinessHour):
         Normalize start/end dates to midnight before generating date range.
     weekmask : str, Default 'Mon Tue Wed Thu Fri'
         Weekmask of valid business days, passed to ``numpy.busdaycalendar``.
+    holidays : list
+        List/array of dates to exclude from the set of valid business days,
+        passed to ``numpy.busdaycalendar``.
+    calendar : np.busdaycalendar
+        Calendar to integrate.
     start : str, time, or list of str/time, default "09:00"
         Start time of your custom business hour in 24h format.
     end : str, time, or list of str/time, default: "17:00"
         End time of your custom business hour in 24h format.
 
     Examples
     --------
-    >>> from datetime import time
+    In the example below the default parameters give the next business hour.
+
     >>> ts = pd.Timestamp(2022, 8, 5, 16)
     >>> ts + pd.offsets.CustomBusinessHour()
     Timestamp('2022-08-08 09:00:00')
+
+    We can also change the start and the end of business hours.
+
+    >>> ts = pd.Timestamp(2022, 8, 5, 16)
     >>> ts + pd.offsets.CustomBusinessHour(start="11:00")
     Timestamp('2022-08-08 11:00:00')
-    >>> ts + pd.offsets.CustomBusinessHour(end=time(19, 0))
+
+    >>> from datetime import time as dt_time
+    >>> ts = pd.Timestamp(2022, 8, 5, 16)
+    >>> ts + pd.offsets.CustomBusinessHour(end=dt_time(19, 0))
     Timestamp('2022-08-05 17:00:00')
-    >>> ts + pd.offsets.CustomBusinessHour(start=[time(9, 0), "20:00"],
-    ...                                    end=["17:00", time(22, 0)])
-    Timestamp('2022-08-05 20:00:00')
+
+    >>> ts = pd.Timestamp(2022, 8, 5, 22)
+    >>> ts + pd.offsets.CustomBusinessHour(end=dt_time(19, 0))
+    Timestamp('2022-08-08 10:00:00')
+
+    In the example below the business hours start at the default value "09:00"
+    and end at "8:00".
+
+    >>> ts = pd.Timestamp(2022, 8, 5, 22)
+    >>> ts + pd.offsets.CustomBusinessHour(end="8:00")
+    Timestamp('2022-08-05 23:00:00')
+
+    >>> pd.offsets.CustomBusinessHour(end="8:00")
+    <CustomBusinessHour: CBH=09:00-08:00>
+
```

Comment:
I'm not really sure what this example demonstrates that's not already shown by the others...would be partial to removing it

Reply:
Thank you. I deleted this example and updated my PR.